### ISSUE TITLE
[Bug] Wrap <p> around <va-link> to support text resizing

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#4752]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

In the `src/applications/simple-forms/shared/components/GetFormHelp.jsx` the `<va-link>` text did not resize when users selected this option setting in their browser.

<img width="677" height="170" alt="Image" src="https://github.com/user-attachments/assets/2e996227-c4f2-45c0-a4b6-6ad6166c664c" />

The text looks like this when that is enabled.

<img width="717" height="286" alt="Image" src="https://github.com/user-attachments/assets/157bdc3d-1238-4f46-836d-b8c7f36b93d2" />

## What I expected to happen

When enlarging the text using browser settings, I expected the "Get help filling out a form" text to enlarge the same as the other text, looking like this:

<img width="674" height="315" alt="Image" src="https://github.com/user-attachments/assets/58da5d50-21de-4d48-a384-ced374bd3114" />

## Reproducing

- CSS Library version:
- Device: (e.g. iPhone 8, Macbook) Backbook
- Browser: (e.g. Firefox, IE 11) Chrome

**URL(s) where this bug occurs**

- http://staging.va.gov/records/request-personal-records-form-20-10206/introduction
- (The bottom of most form introduction pages, and other internal form pages)
-

**Steps to reproduce:**

1. Use browser settings to change font size (this is not the same as zooming in)
2. In chrome, go to chrome://settings/?search=font+size
3. Change font to "Very Large"
4. View page in browser

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [X] This bug is not blocking any work
- [ ] Other

### Details

[This slack thread explains the issue](https://dsva.slack.com/archives/C01DBGX4P45/p1735058549203829)
